### ISSUE TITLE
Warning when running

### ISSUE THREAD

**Issue by Kvaz1r** [nelsam/vidar#32]
When start with arguments new file, I get this warnings:

> E:\gopath\src\github.com\nelsam\vidar>vidar 1.go
> 2016/04/09 13:23:25 Received error reading directory /: open /: Access is denied.

as it should be?

**Comment by nelsam** [comment]
I haven't actually tried to run vidar on a Windows machine, yet; it looks like I need to fix some Windows-specific bugs.  Most of that will be guess-work for me, as I don't have access to a Windows computer, but I'll at least fix some of the file path issues and figure out if any of the libraries I'm using have issues on Windows systems.

**Comment by nelsam** [comment]
I've made changes to be a little more OS-agnostic (hopefully) in cc83709d8ac8de9dcccbf2a471b949e53911b8d5.  I'm hoping this will solve both of the issues you posted, but I'm not yet certain about that.

I'm currently downloading Windows VMs so that I can test it out, but it will be a couple of hours before that's done.

**Comment by Kvaz1r** [comment]
unfortunately, now throws exceptions:

> github.com/nelsam/vidar/navigator.(*TOC).Close(0x0, 0x0, 0x0)
>         E:/gopath/src/github.com/nelsam/vidar/navigator/toc_tree.go:308 +0x2b
> github.com/nelsam/vidar/navigator.(*ProjectTree).SetRoot(0xc0822089c0, 0xc082002c30, 0x2)

**Comment by nelsam** [comment]
Alright, fair enough. I know sort of what's going on, but I have Windows
virtual machines downloaded now. I'll let you know as soon as I have it
fixed.

**Comment by nelsam** [comment]
I finally managed to succeed in setting up a Windows VM with hardware acceleration.  Since `gxui` uses opengl calls, I had to ensure that hardware acceleration was working before I could run the editor.

I've got some of these issues fixed, but there are a few more that I've noticed already - stuff like the "open file" dialog using "/" to mean the root of the filesystem.  I'll continue working on it over the next week.
